platform/android/src/ch/opengis/qfield/QFieldActivity.java

```diff
@@ -173,6 +177,28 @@ public void onNewIntent(Intent intent) {
         }
     }
 
+    @Override
+    public boolean onKeyDown(int keyCode, KeyEvent event) {
+        if (handleVolumeKeys && (keyCode == KeyEvent.KEYCODE_VOLUME_UP ||
+                                 keyCode == KeyEvent.KEYCODE_VOLUME_DOWN ||
+                                 keyCode == KeyEvent.KEYCODE_MUTE)) {
```

Comment:
I find it a bit confusing there is a function `onKeyDown` that has `if keyCode == VOLUME_UP: volumeKeyDown()`. Probably a comment why is that would be nice?